### DISCUSSION TITLE
Sinhala language translation

### DISCUSSION THREAD

**Discussion by vinuthjaya** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1231] [General]
[Default.strings copy.zip](https://github.com/AppHouseKitchen/AlDente-Charge-Limiter/files/15409876/Default.strings.copy.zip)

**Comment by davidwernhart** [comment]
Thank you very much for helping us to localize AlDente in Sinhala! :) We very much appreciate that.

Unfortunately, the translation seems to be incomplete. Please follow the tutorial here to get the current translation file from the app: https://apphousekitchen.com/translate-aldente-and-get-50-off-aldente-pro-for-a-year/

Thank you for your effort and kind regards,

David
